frontend/src/components/CreateRecipeForm/CreateRecipeForm.js

```diff
@@ -4,11 +4,13 @@ import reject from 'lodash/reject';
 import { withRouter } from 'react-router-dom';
 import Select from 'react-select';
 import TagsBar from '../TagsBar/TagsBar';
-import { createRecipe, getTags } from '../../lib/apiClient';
+import {
+  createRecipe, editRecipe, getTags,
+} from '../../lib/apiClient';
 import barChart from '../../assets/bar-chart.png';
 import Footer from '../Footer/Footer';
-import './EditRecipeForm.css';
-import EditDirections from './EditDirections/EditDirections';
+import './CreateRecipeForm.css';
```

Comment:
I'm just curious, any reason the names of everything were changed to `CreateRecipe...`? It feels weird in a PR that changes them to add editing capabilities. :)

Reply:
I changed them when I first started this ticket thinking that I'd need to create a separate component for editing recipes, and the form used to create recipes was called EditRecipeForm.